foundation/foundation-mda/src/main/resources/templates/general-mlflow/training.config.properties.vm

```diff
@@ -5,4 +5,7 @@
 # Originally generated from: ${templateName} 
 
 mlflow_tracking_uri = http://mlflow-ui-tracking:5005
-kafka_server = kafka-cluster:9093
\ No newline at end of file
+kafka_server = kafka-cluster:9093
+#if (${containsOnnx})
+onnx_model_directory = onnx_model
```

Comment:
Q: Was this for completeness sake, or was there an error that's being resolved?

Reply:
Also based on the later usage, it seems like this should be more generic than `onnx_model_directory`. Maybe `converted_model_directory`?

Reply:
This is to address an error with post actions. Discussed in our SU, by previously removing the model_directory and using MLflow tracking server to save artifacts, post actions was unable to locate a path to save the file. Hence, this approach 1) enables downstream projects to save their onnx model to the same location as their model (through MLflow to S3 bucket) 2) provides projects to change the name of the directory saved.